### DISCUSSION TITLE
CouchDB_Import_Demographics.php error

### DISCUSSION THREAD

**Discussion by Johan-Peralta** [aces/Loris#9499] [Q&A]
Hi, I am getting the following error when executing the CouchDB_Import_Demographics.php script on LORIS v26.0.0:

```
/var/www/loris/tools$ php CouchDB_Import_Demographics.php
Updating Config:BaseConfig: unchanged contentsDemographics_Session_HBN0000_VIS1: new
Demographics_Session_DCC0000_VIS1: new
PHP Fatal error:  Uncaught TypeError: Project::getProjectFromID(): Argument #1 ($projectID) must be of type ProjectID, int given, called in /var/www/loris/tools/CouchDB_Import_Demographics.php on line 380 and defined in /var/www/loris/php/libraries/Project.class.inc:105
Stack trace:
#0 /var/www/loris/tools/CouchDB_Import_Demographics.php(380): Project::getProjectFromID()
#1 /var/www/loris/tools/CouchDB_Import_Demographics.php(496): CouchDBDemographicsImporter->_updateDataDict()
#2 /var/www/loris/tools/CouchDB_Import_Demographics.php(512): CouchDBDemographicsImporter->run()
#3 {main}
  thrown in /var/www/loris/php/libraries/Project.class.inc on line 105

```
The projects seem to be properly setup in the MySQL side.
Any ideas on what could be causing the error?

Any help will be greatly appreciated, Johan

**Comment by Johan-Peralta** [comment]
Hi LORIS Community, I wanted to kindly follow up on my earlier question regarding the error encountered when running the CouchDB_Import_Demographics.php script on LORIS v26.0.0.

Steps I’ve tried:
1. Verified that the projects are set up correctly in the MySQL database.
2. Reviewed the getProjectFromID method in Project.class.inc to confirm its argument type requirement (ProjectID).
3. Traced the $projectID value in CouchDB_Import_Demographics.php to check if it is passed as an int.

Could you please confirm if there are any additional configurations required for this script to work correctly in LORIS v26.0.0?

P.S. The CouchDB_Import_Instruments.php script ran fine, which suggests the issue might be specific to the CouchDB_Import_Demographics.php implementation.

Thank you for your time and assistance. Any pointers or documentation references would be greatly appreciated.

Best regards, Johan

**Comment by driusan** [comment]
Resolved at #9507 / #9508
